src/Symfony/Component/Security/Core/Authorization/AccessDecisionManager.php

```diff
@@ -46,13 +47,40 @@ public function __construct(iterable $voters = [], AccessDecisionStrategyInterfa
         $this->strategy = $strategy ?? new AffirmativeStrategy();
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getDecision(TokenInterface $token, array $attributes, mixed $object = null, bool $allowMultipleAttributes = false): AccessDecision
```

Comment:
`$allowMultipleAttributes` is in `decide` method (but not in interface). I think we can remove this arg and the check WDYT ?

Original comment:
`Special case for AccessListener, do not remove the right side of the condition before 6.0`